packages/cli-client/overrides/core-crypto/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@wireapp/core-crypto",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./index.js",
+  "author": "",
+  "license": "ISC"
```

Comment:
Shouldn’t this be gpl3?

Reply:
This is not published anywhere anyway. This is a totally fake package

Reply:
fixed with https://github.com/wireapp/wire-web-packages/pull/5035/commits/78f6c46333d9d01d9db7c161169f1ce35f9b196c